dashboard/src/t5gweb/templates/ui/table.html

```diff
@@ -107,6 +108,7 @@
                                     </td>
                                     <td class="align-middle text-center">{{ new_comments[account][status][card]['case_days_open'] }}</td>
                                     <td class="align-middle text-center">{{ new_comments[account][status][card]['case_updated_date'] }}</td>
+                                    <td class="align-middle text-center">{{ new_comments[account][status][card]['daily_telco'] }}</td>
```

Comment:
I'm guessing this will cause an error until the next time the cache runs and adds that label to all the cards?

Reply:
Looks like flask/jinja catches it. In my dev environment, if I create a new column with a fake key (`<td class="align-middle text-center">{{ new_comments[account][status][card]['test'] }}</td>`), it just creates an empty column:
![Screenshot from 2023-12-01 15-12-35](https://github.com/RHsyseng/t5g-field-support-team-utils/assets/56094214/0eacd82e-4a50-45d2-a17d-3d3a846e2f79). 

In this case the column is hidden, so it should be okay, I think

Reply:
YOLO